docs/README.md

```diff
@@ -126,3 +126,16 @@ To control what API docs get built, you can set any combination of the following
 * `SKIP_RDOC=1`: Skip the R API docs.
 * `SKIP_SQLDOC=1`: Skip the SQL API docs.
 
+## Build docs with docker image (Optional)
+
+As a Spark developer, you can generate all documents locally as follows:
+
+Note: Before running it, you need to have `docker` installed.
+
+```sh
+$ sh dev/spark-test-image-util/docs/build-docs
```

Comment:
sh is not required after granting x permission
```suggestion
$ dev/spark-test-image-util/docs/build-docs
```

Reply:
But it's a bit strange, I granting it as `chmod a+x dev/spark-test-image-util/docs/build-docs`, but it didn't show up in the git logs. Do you know why? thanks.

Reply:
But it's a bit strange, I granting it as `chmod a+x dev/spark-test-image-util/docs/build-docs`, but it didn't show up in the git logs. Do you know why? thanks.
```
(base) ➜  spark-trunk git:(master) ✗ ls -al dev/spark-test-image-util/docs/build-docs
-rw-r--r--@ 1 panbingkun  staff  2794 Nov 29 16:17 dev/spark-test-image-util/docs/build-docs
```

Reply:
It seems to work, let's take another look this time.

Reply:
git add --chmod=+x dev/spark-test-image-util/docs/build-docs